README.md

```diff
@@ -1,2 +1,58 @@
 # publicly-available-microscopy-data
-Compiling and describing publicly available microscopy datasets
+## Purpose
+Compile and describe publicly available microscopy datasets using alpha and beta diversity measures. We extract metadata for each well and compute diveristy statistics for image attributes including but not limited to `screen id`, `genotype`, `phenotype`, `imaging method`, `cell line`, and `organism`. These measures give us insight into the quality of these datasets for use in image-based profiling methods.
+
+## Dependencies
```

Comment:
I recommend deleting this section to avoid having to update two things every time you add a package. (instead, you can say to look to the `environment.yml` or not specify anything)

Reply:
Section was deleted and `environment.yml` was referenced.

Added in 474483c3b8abc549737b3e98b658ef3f0d6d9f22 (removed dependencies) 983a48106ccf8d5145ff1c82c7f890022f5b17db (referenced `environment.yml`).